analyses/gene-set-enrichment-analysis/run-gsea.sh

```diff
@@ -40,6 +45,22 @@ INPUT_FILE="${DATA_DIR}/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds"
 OUTPUT_FILE="${RESULTS_DIR}/gsva_scores_stranded.tsv"
 Rscript --vanilla 01-conduct-gsea-analysis.R --input ${INPUT_FILE} --output ${OUTPUT_FILE}
 
-
 ######## Model GSVA scores ############
+# Only run when pbta-histologies.tsv is generated which has integrated_diagnosis
 Rscript -e "rmarkdown::render('02-model-gsea.Rmd', clean = TRUE, params=list(is_ci = ${IS_CI}))"
+else
+DATA_DIR="../collapse-rnaseq/results"
```

Comment:
I *think the `DATA_DIR` and `RESULTS_DIR` bits are the only parts that change in this if/then, else statement. If so, this can be made a lot shorter by only having these `DATA_DIR` and `RESULTS_DIR` declarations in the if statement and you only need the actual script calls once.

Reply:
oh yeah sounds good, I'll update that

Reply:
ahh I didn't read my own comment :D so the DATA_DIR and RESULTS_DIR bits change as per the condition for subtyping is T/F. but also 02-model-gsea.Rmd only needs to be run when we have the final pbta-histologies.tsv. Which also brings up the #877 issue that this needs to be harmonized_diagnosis in a future PR.

```
# Only run when pbta-histologies.tsv is generated which has integrated_diagnosis
Rscript -e "rmarkdown::render('02-model-gsea.Rmd', clean = TRUE, params=list(is_ci = ${IS_CI}))"
```

Reply:
Okay, but aren't lines 37 - 46 and 55 - 64  and still redundant? I think you can include this rmarkdown call in the T/F statement but still don't need the rest of this repeated.